Packs/IBMResilientSystems/Scripts/IbmConvertCommentsToTable/IbmConvertCommentsToTable.yml

```diff
@@ -0,0 +1,16 @@
+commonfields:
+  id: IbmConvertCommentsToTable
+  version: -1
+comment: This script is used to convert IBM QRadar SOAR notes to a markdown table.
```

Comment:
Should this be comments?
This script is used to convert IBM QRadar SOAR comments to a markdown table.